xgcm/test/test_transform.py

```diff
@@ -812,7 +812,7 @@ def test_conservative_interp_warn():
 
 @pytest.mark.skipif(numba is None, reason="numba required")
 def test_grid_transform_noname_data(multidim_cases):
-    """ Check handling of a `da` input without name"""
```

Comment:
I guess this is just due to an updated black version?

Reply:
Yes. I updated the pre-commit hook version